typescript/forms.d.ts

```diff
@@ -49,6 +49,11 @@ export type FormElementBase = _FormElementBase & {
    * the element label.
    */
   hint?: string
```

Comment:
Can we also update the doc for this property to describe the relationship between the two properties